gcc/config/riscv/riscv.h

```diff
@@ -22,6 +22,7 @@ along with GCC; see the file COPYING3.  If not see
 #ifndef GCC_RISCV_H
 #define GCC_RISCV_H
 
+#include <stdbool.h>
```

Comment:
I tried to remove this, but it reports error during compilation. Because I am using "bool used_vrs[MAX_ARGS_IN_VECTOR_REGISTERS]" in the CUMULATIVE_ARGS structure. This variable is used to handle function arg of RVV vector register. Do you want me to change it into "int used_vrs[MAX_ARGS_IN_VECTOR_REGISTERS]"?

Reply:
Oh this file has included during compiling libgcc, and that's in C mode..., okay, let keep that!

Reply:
> 

I saw your comments. I agree these issues should be fixed. Thanks for that. Let me finish splitting the patch and upload the rest of the codes. Then I will fix the issues one by one as you mention.